packages/connect-explorer/src/reducers/trezorConnectReducer.ts

```diff
@@ -46,6 +46,22 @@ const removeDevice = (state: ConnectState, device: TrezorConnectDevice): Connect
     return state;
 };
 
+const onOptionChange = (state: ConnectState, field: Field<any>, value: any): ConnectState => {
```

Comment:
low priority: maybe value could be typed better?  and maybe field as well 🤔

Reply:
1e33ca1a0731c1c9591385bf1c913359a67e03cb